clients/typescript/src/util/common.ts

```diff
@@ -38,6 +40,23 @@ export const base64 = {
 
 export const DEFAULT_LOG_POS = numberToBytes(0)
 
+const trueByte = 116 // the character 't'
+const falseByte = 102 // the character 'f'
```

Comment:
```suggestion
const trueByte = 't'.charCodeAt(0)
const falseByte = 'f'.charCodeAt(0)
```

I slightly prefer deriving the constant as to avoid the typos